app/src/test/java/org/mozilla/focus/session/SessionCallbackProxyTest.kt

```diff
@@ -12,6 +12,8 @@ import org.mockito.Mockito.mock
 import org.mozilla.focus.iwebview.IWebView
 import org.robolectric.RobolectricTestRunner
 
+private val TEST_URL = "https://github.com/mozilla-mobile/focus-android/"
```

Comment:
nit: `private const val` :P I'm not sure what the value of `const` is though, tbh... compiler note?

Reply:
Was curious and decompiled to Java:

WIthout `const`:

```Java
public final class SessionCallbackProxyTestKt {
   private static final String TEST_URL = "https://github.com/mozilla-mobile/focus-android/";

   // $FF: synthetic method
   @NotNull
   public static final String access$getTEST_URL$p() {
      return TEST_URL;
   }
}
```

With `const`:

```Java
public final class SessionCallbackProxyTestKt {
   private static final String TEST_URL = "https://github.com/mozilla-mobile/focus-android/";
}
```